pkg/env/deployment_enhancement.go

```diff
@@ -6,4 +6,8 @@ var (
 	// CentralDeploymentEnhancementTimeout allows to configure the time Central waits for Sensor to answer to a
 	// DeploymentEnhancementRequest.
 	CentralDeploymentEnhancementTimeout = registerDurationSetting("ROX_CENTRAL_DEPLOYMENT_ENHANCE_TIMEOUT", 30*time.Second)
+
+	// SensorEnhancementQueueSize configures the size of the buffered channel that incoming
+	// DeploymentEnhancementRequests are queued in
+	SensorEnhancementQueueSize = RegisterIntegerSetting("ROX_SENSOR_ENHANCEMENT_QUEUE_SIZE", 50)
```

Comment:
[nit]: Maybe `SensorDeploymentEnhancementQueueSize` would be better? Somehow I think that leaving `Deployment` out of the name might be confusing out of context.
```suggestion
	SensorDeploymentEnhancementQueueSize = RegisterIntegerSetting("ROX_SENSOR_DEPLOYMENT_ENHANCEMENT_QUEUE_SIZE", 50)
```

Reply:
That's a very good point, thanks!